### ISSUE TITLE
Print Format

### ISSUE THREAD

**Issue by TimLiu98** [ManuelPalermo/X-Ray-ConvNet#3]
Hi, bro, I am here again, I have a question about this print:

Print Content:  - 355s - loss: 0.6117 - acc: 0.6788 - val_loss: 0.4495 - val_acc: 0.6802

This is just an example, not that Print Content:  is added by myself, this line is printed each epoch, I wonder if I can change the print format of this part, for example:

time: 355s 
loss: 0.6117 
acc: 0.6788 
val_loss: 0.4495 
val_acc: 0.6802

I take a long time to find all codes for printing but without this line

Btw, when I run the code, the result are always:

val_loss did not improve from 0.44251

Sometimes from the start to the earlystop, is it the same as your result or my parameters are not good?

**Comment by ManuelPalermo** [comment]
Welcome back,

This project is just a toy example on how to use DeepLearning(and I was very limited on compute), as such the performance was not that great(don't remember exactly the results, but are probably similar to what you are getting). The default parameters don't produce great results, some ideas to improve:
* use all the training data available
* dont use clahe when pre-processing database as it can introduce artifacts not present in original image(just resize)
* use bigger image shape(resize to 128x128 might be loosing important information - try at least 256 or 512)
* load a pretrained backbone model instead of using the "custom" one
* use global average pooling on the classification head instead of flatten
* decrease the augmentations as these might be corrupting the images to a point that there is no information to correctly classify its class.
* (train on google colab if you have a weak computer like mine)

Let me know if the results improve.

**Comment by TimLiu98** [comment]
Thanks bro, I will try it when I am free, so the format cannot be changed?

**Comment by ManuelPalermo** [comment]
You can add some "\n" to the print where you want to break the line. You just need to find where the print is done and add that. If the print is made inside keras(I dont remember) you might need to hack it a bit to change it.

**Comment by TimLiu98** [comment]
Thanks bro, I am trying to find it now, once there are any performance improvement, i will let you know.

**Comment by TimLiu98** [comment]
I had tried some of them, but how to understand 

# execute 0 to 5 of the following (less important) augmenters per image
# don't execute all of them, as that would often be way too strong

Just comment on some content in iaa.SomeOf? Which parts should I keep with experience, or I must try one by one?

**Comment by TimLiu98** [comment]
I had tried some of them, but how to understand 

# execute 0 to 5 of the following (less important) augmenters per image
# don't execute all of them, as that would often be way too strong

Just comment on some content in iaa.SomeOf? Which parts should I keep with experience, or I must try one by one?

**Comment by TimLiu98** [comment]
I had tried some of them, but how to understand 

execute 0 to 5 of the following (less important) augmenters per image
don't execute all of them, as that would often be way too strong

Just comment on some content in iaa.SomeOf? Which parts should I keep with experience, or I must try one by one?
